### DISCUSSION TITLE
Problem in Formkit Schema Text Field

### DISCUSSION THREAD

**Discussion by KaranKharabe** [formkit/formkit#1469] [General]
- I have to upperCase  Text field of formkit schema how i can do 

below Code i use but its not work 

`<script setup>
import { ref, reactive } from 'vue'
import { FormKitSchema } from '@formkit/vue' // Ensure you have the correct path for FormKitSchema

const workflow = ref('') // Initialize with an empty string or appropriate default value

const schema = [
  {
    $formkit: 'text',
    name: 'workflow',
    bind: '$someAttributes',
  },
]

const data = reactive({
  someAttributes: {
    onInput: (event) => {
      workflow.value = event.target.value
    }
  }
})
</script>

<template>
  <FormKitSchema :schema="schema" :data="data" />
</template>
`

**Comment by sashamilenkovic** [comment]
I would use a plugin and an input hook: https://formkit.link/37dbfaec28ace9c885a33619ba1f50e5

Information on hooks can be found here: https://formkit.com/essentials/architecture#hooks
